src/test/xattr_bench.cc

```diff
@@ -152,15 +152,6 @@ int main(int argc, char **argv) {
   argv_to_vec(argc, (const char **)argv, args);
 
   global_init(0, args, CEPH_ENTITY_TYPE_CLIENT, CODE_ENVIRONMENT_UTILITY, 0);
-  common_init_finish(g_ceph_context);
```

Comment:
Thank You @tchaikov for your nice explanation, got your point. Will change it back.